sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -284,9 +284,9 @@ revamped configuration API. Static `create_config` methods have been renamed
 
 ### New features:
 - Added credentials for authenticating users:
-[`DeviceCodeCredential`](https://azure.github.io/azure-sdk-for-python/ref/azure.identity.html#azure.identity.DeviceCodeCredential),
-[`InteractiveBrowserCredential`](https://azure.github.io/azure-sdk-for-python/ref/azure.identity.html#azure.identity.InteractiveBrowserCredential),
-[`UsernamePasswordCredential`](https://azure.github.io/azure-sdk-for-python/ref/azure.identity.html#azure.identity.UsernamePasswordCredential)
+ - `DeviceCodeCredential`
```

Comment:
@chlowell can you sign of of that?

Reply:
Works for me, we don't have docs for this version (1.0.0b2) anymore and I prefer removing these to adding links to later documentation.